File: Modul-4/Soal-5/PRAK405-2410817220030-RACHELWINAYUDA.c
#include <stdio.h>

int hitung_kelipatan(int n, int kelipatan) {
    int total = 0;
    for (int i = 1; i <= n; i++) {
        int baris_total = 0;
        printf("(");
        for (int j = i; j >= 1; j--) {
            baris_total += j * kelipatan;
            printf("%d * %d", j, kelipatan);
            if (j > 1) {
                printf(") + (");
            }
        }
        printf(") = %d\n", baris_total);
        total += baris_total;
    }
    return total;
}

int main() {
    int n, kelipatan;
    scanf("%d", &n);
    scanf("%d", &kelipatan);
    int total = hitung_kelipatan(n, kelipatan);
    printf("%d\n", total);
    return 0;
}

File: Modul-4/Soal-5/PRAK405-2410817220030-RACHELWINAYUDA.py
input_angka = input()
n, kelipatan = map(int, input_angka.split())
total = 0

for i in range(1, n + 1):
    hasil = 0
    for j in range(i, 0, -1):
        hasil += j * kelipatan
        print(f"({j} * {kelipatan})", end="")
        if j > 1:
            print(" + ", end="")
    print(f" = {hasil}")
    total += hasil
print(total)